shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/ConferenceService.kt

```diff
@@ -34,6 +34,9 @@ class ConferenceService(
     private var userId2024: String? by storage.bind(String.serializer().nullable) { null }
     private var needsOnboarding: Boolean by storage.bind(Boolean.serializer()) { true }
     private var notificationsAllowed: Boolean by storage.bind(Boolean.serializer()) { false }
+    private var _themeStorage: Theme by storage.bind(Theme.serializer()) { Theme.SYSTEM }
+    private val _theme = MutableStateFlow(_themeStorage)
```

Comment:
These can be consolidated later if we set up DataStore for storage, which always exposes the values as Flows anyway